README.md

```diff
@@ -1,13 +1,14 @@
 # knapsack
-Pseudo-polynomial approach to Knapsack Problem
 
-I thought I might write one algorithm that I find really interesting: Knapsack problem.
-
-Its problem statement is simple:
+Knapsack is a Pseudo-polynomial approach to Knapsack Problem. Knapsack problem is a problem, which I personally interest me. I decided to write an algorithm to solve the problem.
 
+The knapsack problem has a fixed size knapsack. 
```

Comment:
- **Not true**.  The Knapsack problem has many variants.  Among them is the 0-1 Knapsack, which is the problem in question, here.